wire-ios/Wire-iOS/Sources/UserInterface/Company Login/CompanyLoginController.swift

```diff
@@ -330,9 +330,14 @@ extension CompanyLoginController {
             case let .success(backendEnvironment):
                 requestUserConfirmationForBackendSwitch(to: backendEnvironment) { didConfirm in
                     guard didConfirm else { return }
-                    sessionManager.switchBackend(to: backendEnvironment)
-                    BackendEnvironment.shared = backendEnvironment
-                    self.startAutomaticSSOFlow(promptOnError: false)
+                    sessionManager.switchBackend(to: backendEnvironment) { error in
+                        if let error {
+                            self.presentCompanyLoginAlert(error: .unknown)
```

Comment:
please log the error, otherwise it's hard to know what happened

Reply:
done 0f79611f0bced5b89077712d8b9158abffd0cb76